psl-evaluation/src/main/java/org/linqs/psl/utils/evaluation/statistics/DiscretePredictionComparator.java

```diff
@@ -78,7 +78,7 @@ public void setResultFilter(AtomFilter af) {
 	@Override
```

Comment:
Did "Also, change the if on line 144 to a negative condition and continue if it gets in. ". Part 1 raised as separate issue.